### ISSUE TITLE
Can not Get the Receiving Updates via MQTT

### ISSUE THREAD

**Issue by Abouzeid** [FraunhoferIOSB/FROST-Server#69]
Hi all, 

When I change the description of a Thing, I get the notification on the 'FROST-Bus' topic not on "v1.0/Thigns".

The last Message in the SensorThings.log file. 
`[MqttManager ObservationCreateEventProcessor-Thread] INFO  d.f.iosb.ilt.sta.mqtt.MqttManager - 
 received message on topic 'FROST-Bus' which is no valid topic to create an observation.`

I can successfully create an observation via MQTT and get the notification on "v1.0/Observations" topic. So why I am getting the above message when I update a thing's property. Do I miss something?

Lastly, it's a bit off topic but I'm just wondering why do we need PostGIS Extension to the Postgresql?

Thank you in advance.

Any help will be appreciated.

**Comment by hylkevds** [comment]
If you see anything on the 'FROST-Bus' topic then I suspect you're either connected to the wrong MQTT server, or you've configured the server wrong and are trying to use the client-facing MQTT server as internal message bus.

If you have a look at the second architecture image: https://github.com/FraunhoferIOSB/FROST-Server/blob/master/docs/architecture-packages.adoc#img-arch-multi
You can see MQTT is used in two locations, as an internal message bus, and client-facing using the STA standard.

If you're getting messages on "v1.0/Observations" then you should also be getting messages on "v1.0/Things" (assuming the typo you made in "v1.0/Thigns" is just in your post here)

How are you running the server?

**Comment by hylkevds** [comment]
As to why you need PostGIS: It's to enable the geospatial queries that are defined in the SensorThings API: http://docs.opengeospatial.org/is/15-078r6/15-078r6.html#56

**Comment by Abouzeid** [comment]
HI @hylkevds 

Thanks for your answer.
- The typo is just in my post. (sorry for that)

How am I running the server?
After I had built the project, I only got two .war files. 
Therefore,  I'm running the FROST-Server on a Tomcat with two servlets FROST-Server.**HTTP**-1.8-SNAPSHOT  and FROST-Server.**MQTTP**-1.8-SNAPSHOT on the same machine.

MQTTP Config file:
"mqtt.internalHost" value="192.168.xx.xx"    //the same IP address as the ServiceRoot
"mqtt.Host" value="0.0.0.0"


    <Parameter override="false" name="bus.busImplementationClass" value="de.fraunhofer.iosb.ilt.sta.messagebus.InternalMessageBus" description="The java class used for connecting to the message bus."/>
